templates/index2.html

```diff
@@ -292,84 +292,68 @@ <h3>Featured snaps</h3>
       <div class="row">
         <div class="col-3">
           <div class="p-heading-icon">
-            <div class="p-heading-icon__header">
-              <a class="p-heading-icon__img" href="/atom">
-                <img src="https://assets.ubuntu.com/v1/694b2e80-atom.png" alt="" />
-              </a>
-              <h5 class="u-no-margin"><a href="/atom">Atom</a></h5>
-            </div>
+            <a class="p-heading-icon__header" href="/atom">
```

Comment:
Shouldn't this be generated from the same feed as the `/discover` page?

Reply:
Home page was designed with hardcoded list of featured snaps.

Yes, it seems it could make sense to reuse the same feed on home page as well.

Reply:
@bartaz please could you do that? I don't think we should have two different lists of "featured" snaps on the same site.